lib/virtual-fs/src/mem_fs/file.rs

```diff
@@ -144,6 +144,30 @@ impl VirtualFile for FileHandle {
         node.metadata().created
     }
 
+    fn set_accessed(&mut self, atime: u64) {
+        let mut fs = match self.filesystem.inner.write() {
+            Ok(fs) => fs,
+            _ => return,
+        };
+
+        let inode = fs.storage.get_mut(self.inode);
+        if let Some(node) = inode {
+            node.metadata_mut().accessed = atime;
+        }
+    }
+
+    fn set_modified(&mut self, mtime: u64) {
+        let mut fs = match self.filesystem.inner.write() {
+            Ok(fs) => fs,
+            _ => return,
+        };
+
+        let inode = fs.storage.get_mut(self.inode);
+        if let Some(node) = inode {
+            node.metadata_mut().modified = mtime;
+        }
+    }
+
```

Comment:
We should unite this two in one function, since the systemcall has two functions at the beginning.
Please see: https://github.com/bytecodealliance/cap-std/blob/8f2f60f48f39e2b1606ae2961a5fe935dce75ab9/cap-primitives/src/rustix/freebsd/fs/set_times_impl.rs#L9

Reply:
I agree.